EKF/ekf_helper.cpp

```diff
@@ -824,27 +824,43 @@ void Ekf::get_ekf_ctrl_limits(float *vxy_max, float *vz_max, float *hagl_min, fl
 
 }
 
-bool Ekf::reset_imu_bias()
+void Ekf::resetIMUBias()
 {
-	if (_imu_sample_delayed.time_us - _last_imu_bias_cov_reset_us < (uint64_t)10e6) {
```

Comment:
was this not needed?

Reply:
I'd rather be in control of it firmware side and prevent rapid switching.

Reply:
ok